Automattic/wc-calypso-bridge
# Fix icon colors on the Extensions > Discover lading page

### Changes proposed in this Pull Request:



This PR uses admin color CSS variables with the RGB format. **Do not merge**, until the following PR in Jetpack is merged and released: https://github.com/Automattic/jetpack/pull/29304

Example result with the _Modern_ color scheme:
![](https://cdn-std.droplr.net/files/acc_1208648/Rx1q2v)


Closes #.






### How to test the changes in this Pull Request:




1. Notice the icons in the Extensions > Discover page while in ecom free trial.
2. Change your admin color scheme.
3. Navigate to the Extensions > Discover page again, and notice that icons are cohesive to the primary color.



### Other information:




### FOR PR REVIEWER ONLY:
